Lambda-School-Labs/mydish-be
# Update/unit to accept any string

# Purpose 

This PR makes a new migration to remove `abbreviations` and `type` from the units table. It also allows users to create recipes with any unit, not just the existing ones in the database. 

# To Test 

Run `knex migrate:latest` (columns `type` and `abbreviation` should no longer be there)
Try adding a recipe with a unit that is not currently existing in the database. 
Try updating that recipe with another new unit. 
Try to get the recipe by its ID.
